docs/src/conf.py

```diff
@@ -93,6 +107,7 @@ def setup(app):
     "numpy": ("https://numpy.org/doc/stable/", None),
     "torch": ("https://pytorch.org/docs/stable/", None),
     "rascaline": ("https://luthaf.fr/rascaline/latest/", None),
+    "ase": ("https://wiki.fysik.dtu.dk/ase/", None),
```

Comment:
I don't think we link to matplotlib docs, but we do link to ASE's.

Reply:
But we use matplotlib commands in the tutorial i.e:

```python
fig, ax = plt.subplots(1, 2, figsize=(8, 5))

ax[0].plot(range(800), potential_energy, label="potential energy")
ax[0].plot(range(800), kinetic_energy, label="kinetic energy")
ax[0].plot(range(800), total_energy, label="total energy")
ax[0].legend()
ax[0].set_xlabel("t / fs")
ax[0].set_ylabel("energy / eV")

ax[1].plot(range(800), temperature)
ax[1].set_xlabel("t / fs")
ax[1].set_ylabel("temperature / K")

fig.tight_layout()
fig.show()
```

Even if matplotlib is not a direct dependency it now comes with ase and since we use the commands we should also link them to the docs.

Reply:
Right, but then I need to add these to the sphinx conf used for gallery generation instead.

Reply:
So I can't get sphinx-gallery to generate any of the links inside the examples, and it seems this is also broken in rascaline: https://luthaf.fr/rascaline/latest/how-to/property-selection.html

We may want to try to fix this, but I don't feel it is central to the tutorials, so I would leave it for later.

Reply:
It works in the usage example though...

https://luthaf.fr/rascaline/latest/examples/first-calculation.html#the-dataset

Reply:
Okay very cool!